crates/net/discv4/src/config.rs

```diff
@@ -247,6 +253,21 @@ impl Discv4ConfigBuilder {
         self
     }
 
+    /// Adds multiple preferred nodes
+    pub fn add_preferred_nodes(
```

Comment:
Let's use consistent naming, if we're calling them trusted in the vars let's call them trusted here